samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -229,8 +243,8 @@ public TestRunner addOutputStream(InMemoryOutputDescriptor streamDescriptor, int
     factory
         .getAdmin(streamDescriptor.getSystemName(), config)
         .createStream(spec);
-    addConfigs(streamDescriptor.toConfig());
-    addConfigs(streamDescriptor.getSystemDescriptor().toConfig());
+    streamDescriptor.toConfig().forEach(this.configs::putIfAbsent);
```

Comment:
minor: add a comment here to explain why we use putIfAbsent, instead of overwriting.

Reply:
Using the confix prefix, added a comment to docs